tasties_app/models.py

```diff
@@ -26,6 +38,46 @@ class Recipe(models.Model):
     minutes_to_make = models.IntegerField(validators=[MinValueValidator(1)])
     recipe_picture = models.ImageField(upload_to="recipe_pictures")
 
+    def edit_recipe(self, new_title, new_description,
```

Comment:
I think you should give the parameters their current values as default values. When we edit a recipe, we won't always change EVERY field. Sure, we can give the recipe its current field values as parameters in this method upon calling it, but that could be a bit of a mess.

Reply:
I think Efraim right, I think you should do the default values to the parameters so we can edit recipe as we want and not have to give all the parameters.

Reply:
I think Efraim right, I think you should do the default values to the parameters so we can edit recipe as we want and not have to give all the parameters.